src/components/Directory/MentorCard/MentorCard.js

```diff
@@ -0,0 +1,112 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import useStyles from './ MentorCard.styles';
+import Card from '@material-ui/core/Card';
+import CardHeader from '@material-ui/core/CardHeader';
+import Divider from '@material-ui/core/Divider';
+import Avatar from '@material-ui/core/Avatar';
+import Button from '@material-ui/core/Button';
+import CardContent from '@material-ui/core/CardContent';
+import Typography from '@material-ui/core/Typography';
+import useMediaQuery from '@material-ui/core/useMediaQuery';
+
+const MentorCard = ({
+  firstName,
+  lastName,
+  title,
+  mentorSkills,
+  bio,
+  initials,
+  onTestClick,
+}) => {
+  // useEffect(() => {
+  //   alert('making fetch Request');
+  // });
+
+  const isMobile = useMediaQuery('(max-width:767px)');
+  const classes = useStyles({ isMobile });
+  const {
+    root,
+    avatar,
+    booking,
+    skillList,
+    jobField,
+    mentorNameField,
+    bioSection,
+  } = classes;
+
+  const handleClick = (e) => {
+    e.preventDefault();
+    if (onTestClick) return onTestClick();
+    // open modal for scheduling;
+    alert('BOOKING WILL BE SCHEDULED');
```

Comment:
let's replaced this with console log, i updated the eslint rule to allow console log, we'll change this later when that component is ready